src/Components/UI/Alert/Alert.jsx

```diff
@@ -0,0 +1,21 @@
+import { useAlert } from "../../../CustomHooks/";
+import "./Alert.css";
+export const Alert = () => {
+  const {
+    showAlert: { alertMessage, type },
+  } = useAlert();
+  const alertClass = `alert-${type}`;
+
+  return (
```

Comment:
You could have used that variable directly inside the className instead of defining a variable.